conf/defaults.ini

```diff
@@ -304,6 +304,9 @@ password_hint = password
 # Default UI theme ("dark" or "light")
 default_theme = dark
 
+# Path to a custom home page. It should match a route in frontend routes and should contain a leading backslash
+home_page =
+
```

Comment:
this needs to be added to to sample.ini as well  (but commented out there), and added to configuration.md docs file